### ISSUE TITLE
Project Dead?

### ISSUE THREAD

**Issue by hroma** [akalongman/sublimetext-codeformatter#423]
ST4 has been released a while and this project didn't get any updates since October 2019....

phpF: This project has been abandoned due to lack of interest. [You may use other forks of this](https://github.com/subins2000/phpF/network).

So, my question is : is this project dead?
Are there any alternatives to this project?

**Comment by edukah** [comment]
+1

**Comment by akalongman** [comment]
Does anyone still use Sublime Text? I suppose everyone moved to VsCode and PHPStorm

**Comment by hroma** [comment]
@akalongman Thanks for confirming that the project is dead. Would be great to archive the project and update the Readme.md accordingly.

**Comment by vhbui02** [comment]
Well, i'm switching from VSCode to SL4 since it's so so so much faster on my Ubuntu VM. VSCode is super laggy when working with projects that have multiple files.

**Comment by ejohann** [comment]
I am using SublimeText, I like that it is fairly lightweight, after all one just wants a code editor right? 

Anyways it will be nice if the formatter supported Perl.

**Comment by Teddielike** [comment]
> Does anyone still use Sublime Text? I suppose everyone moved to VsCode and PHPStorm

Nope... I love Sublime Text. ❤️ (I have tested many different programs… and I love ST more as before... 🤷🏻‍♂️)

**Comment by FlavioSuarez** [comment]
Me too! I don't change ST4 for any other editor...
